internal/controller/vulnerabilityreports_controller.go

```diff
@@ -152,6 +167,23 @@ func (r *VulnerabilityReportReconciler) getObjectFromOwnerReference(ctx context.
 }
 
 func (r *VulnerabilityReportReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	logger := log.FromContext(r.Ctx)
+	r.reports = cmap.New[console.VulnerabilityReportAttributes]()
+	err := helpers.BackgroundPollUntilContextCancel(r.Ctx, 1*time.Minute, false, true, func(_ context.Context) (done bool, err error) {
```

Comment:
make this poll interval 10 minutes i think

Reply:
done